src/main/java/org/moera/node/text/TextConverter.java

```diff
@@ -18,7 +18,7 @@ private String toHtml(SourceFormat format, String source) {
             case PLAIN_TEXT:
                 return PlainTextConverter.toHtml(source);
             case HTML:
-                return source;
+                return HtmlProcessor.process(source);
             case MARKDOWN:
                 return markdownConverter.toHtml(source);
```

Comment:
Why not to add the same postprocessing for Markdown? `<spoiler>` tag would be useful here too.

Reply:
It wouldn't. All html tags in markdown are interpreted as block and spoiler should be inline. I'll just add `||spoiler||` to markdown with one more PR.

Reply:
No. There is a special list of block tags (see `MarkdownConverted.htmlBlockTags()`), all other tags are inline.